integrations/airflow/documentation.yaml

```diff
@@ -0,0 +1,68 @@
+exporter_type: included
+app_name_short: Airflow
+app_name: Apache {{app_name_short}}
+app_site_name: Airflow
+app_site_url: https://airflow.apache.org/
+exporter_name: the Airflow exporter
+exporter_pkg_name: airflow
+exporter_repo_url: https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/metrics.html
+additional_install_info: |
+  To determine if {{exporter_name}} is exposing metrics, access the metrics endpoint
+  with port forwarding and curl.
```

Comment:
by using port forwarding and `curl` as follows:

(This way, we're telling them that we're gonna tell them *how* to do this, not just that they should do it.)